backend/resource2.py

```diff
@@ -35,10 +40,8 @@ def __init__(self, res_id, resource_handler):
         self.remote_folder = RemoteFolder(self.hs, self.res_id)
         self.local_folder = LocalFolder()
 
-        self.path_prefix = self.output_folder + "/" + self.res_id + "/" + self.res_id + "/data/contents/"
+        self.path_prefix = Path(self.output_folder) / self.res_id / self.res_id / 'data' / 'contents'
```

Comment:
Do you not need quotes around the slashes?

Reply:
Not with pathlib. The `/` operator is overridden to join path components (so long as the first component is a `pathlib.Path`).